cnv/cnv_users_guide/cnv-cdi-scratch-space.adoc

```diff
@@ -0,0 +1,20 @@
+[id="cnv-cdi-scratch-space"]
+= CDI scratch space
+include::modules/cnv-document-attributes.adoc[]
+:context: cnv-cdi-scratch-space
+toc::[]
+
+include::modules/cnv-about-datavolumes.adoc[leveloffset=+1]
```

Comment:
The first 2 links are in the same line at are pretty tight, maybe separate them into two lines?

Reply:
The build tool (asciibinder) doesn't mind if there's a line between `include`s or not but I can add a line if you think it makes it more human readable